src/main/java/core/basesyntax/strategy/TransactionStrategyImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+
+public class TransactionStrategyImpl implements TransactionStrategy {
+    @Override
+    public Integer balanceUpdater(String fruitName, int quantity) {
+        Storage storage = new Storage();
```

Comment:
You should have only one object of storage for all your app. Also a greate idea is to add DAO layer to provide access to the storage